src/ui/server/routes/redirects/index.ts

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import { INSURANCE_ROUTES } from '../../constants/routes/insurance';
+import redirectGet from '../../controllers/redirects';
+
+const { MVP_INSURANCE_ROOT } = INSURANCE_ROUTES;
+
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
do we need this if we have ts-ignore?

Reply:
Good shout, we had this in a few routes - no longer required. Removed all instances